controllers/api/project-routes.js

```diff
@@ -0,0 +1,83 @@
+const router = require('express').Router();
+const {User, Issue, Project, Role, Team, User_State, Issue_State, Project_State, Issue_User} = require('../../models');
+
+// GET all projects - /api/project
+router.get('/', (req, res) => {
+    Project.findAll()
+    .then(projectData => res.json(projectData))
+    .catch(err => {
+        console.log(err);
+        res.status(500).json(err);
+    });
+});
+
+// Get single project - /api/project/:id
+router.get('/:id', (req, res) => {
+    Project.findOne({
+        where: {
+            id: req.params.id
+        }
+    })
+    .then(projectData => res.json(projectData))
+    .catch(err => {
+        console.log(err);
+        res.status(500).json(err);
+    });
+});
+
+// Create project - /api/project
+router.post('/', (req, res) => {
+    Project.create({
+        name: req.body.name,
+        github_repo_name: req.body.github_repo_name,
+        github_username: req.body.github_username
+    })
+    .then(projectData => res.json(projectData))
+    .catch(err => {
+        console.log(err);
+        res.status(500).json(err);
+    });
+});
+
+// Update project - /api/project/:id
+router.put('/:id', (req, res) => {
+    Project.update(
+        {
+            name: req.body.name,
+            github_repo_name: req.body.github_repo_name,
+            github_username: req.body.github_username
```

Comment:
username is not required. I think that based on our GitHub research we should get rid of this field on the project table